caper/templates/pages/project.html

```diff
@@ -204,10 +228,11 @@ <h4 class="mb-0">
                         </thead>
                         <tbody>
                             {% for version in prev_versions %}
+                                
                             <tr>
                                 <td scope="row">{{ version.date }}</td>
                                 <td scope="row">Project ID: {{ version.linkid }}
-                                    {% if version.linkid == project.linkid %}
```

Comment:
Yes that makes sense. I set it as the first element because it was the "current" project, and the first element will always be the current project since I sorted it by date. 

It would make sense to change it to checking using version ID, and changing the text on the table to say "Currently Viewing".